torch/onnx/utils.py

```diff
@@ -396,6 +396,9 @@ def _export(model, args, f, export_params=True, verbose=False, training=False,
                 operator_export_type = OperatorExportTypes.ONNX_ATEN_FALLBACK
             else:
                 operator_export_type = OperatorExportTypes.ONNX
+        if operator_export_type == OperatorExportTypes.ONNX_ATEN_FALLBACK:
```

Comment:
We should at least leave a warning message if we are going to ignore user setting and turn this false. Also, it is good practice to not mutate the input arguments themselves as it can be potentially lead to issues if this arguments gets used in the code downstream, e.g. it can happen some new code gets added  below. 

See handling of `keep_initiliazers_as_input` argument below using `_decide_keep_init_as_input`.

Reply:
Sure, makes sense. Now that we have three such arguments that we have branched similarly, I think we can combine the three to avoid duplicate code.

Reply:
Yes, extract the common part into one helper function would be great.